bb_wrapper/models/pagamentos.py

```diff
@@ -74,23 +86,59 @@ def _set_data(cls, values):
         from ..services.pix import PixService
 
         key = values.get("chave")
-
         key_type = PixService().identify_key_type(key)
-        values["formaIdentificacao"] = key_type
+        values["formaIdentificacao"] = key_type.value
 
         if key_type == TipoChavePIX.telefone:
-            values["dddTelefone"] = int(key[:2])
-            values["telefone"] = int(key[2:])
+            values["dddTelefone"] = key[:2]
+            values["telefone"] = key[2:]
         elif key_type == TipoChavePIX.email:
+            PixService().verify_email(key)
             values["email"] = key
         elif key_type == TipoChavePIX.uuid:
             values["identificacaoAleatoria"] = key
         elif key_type == TipoChavePIX.documento:
-            key_value = cpfcnpj.clear_punctuation(key)
-            if len(key_value) == 1:
-                values["cpf"] = int(key_value)
-            else:
-                values["cnpj"] = int(key_value)
+            PixService().verify_document(key, values)
+
+        documento = values.pop("documento", None)
+        if documento is not None:
+            PixService().verify_document(documento, values)
+
+        values.pop("chave")
+        return values
+
+
+class TransferenciaDadosBancariosPIX(BaseModel):
+    data: str
+    valor: float
+    documento: str
+    tipoConta: TipoContaPIX
+    agencia: Optional[str]
+    conta: Optional[str]
+    digitoVerificadorConta: Optional[str]
+    formaIdentificacao: Optional[str]
+    descricaoPagamento: Optional[str]
+    cpf: Optional[str]
+    cnpj: Optional[str]
+    contaPagamento: Optional[str]
+    numeroISPB: Optional[str]
```

Comment:
Não é um dado opcional! Pois é necessário OU o numeroCompe ou o numeroISPB.

Mas um dos dois é necessário. E eu diria que o mais comum é o número ISPB mesmo. Ou seja, vamos exigir sempre o numeroISPB obrigatoriamente.

```suggestion
    numeroISPB: int
```